### ISSUE TITLE
Proptypes killswitch / error suppression

### ISSUE THREAD

**Issue by nruhe** [facebook/prop-types#266]
Hello! I'm currently working in a code base that is mid-refactor. Unfortunately, we've used propTypes poorly in the past and my console is being swarmed with warnings that are obfuscating important debug errors. Is there any way to suppress all propType warnings temporarily without setting the env to prod?

**Comment by ljharb** [comment]
Nope. My suggestion is to, one at a time, fix the propType warnings, until they're gone.
